extensions/binding/org.eclipse.smarthome.binding.bosesoundtouch/META-INF/MANIFEST.MF

```diff
@@ -34,6 +34,7 @@ Import-Package:
  org.eclipse.smarthome.core.thing.type,
  org.eclipse.smarthome.core.types,
  org.eclipse.smarthome.io.net.http,
+ org.osgi.framework,
```

Comment:
In the light of your last discussion in #6337 I had lol when I saw this import :-)
Please remove.

Reply:
This was actually required by eclipse in order to compile the binding. Anyway - will remove it.

Reply:
Done